roles/idm/templates/keycloak.json.j2

```diff
@@ -0,0 +1,12 @@
+{
+  "apiVersion": "aerogear.org/v1alpha1",
+  "kind": "Keycloak",
+  "metadata": {
+    "name": "rhsso"
+  },
+  "spec": {
+    "adminCredentials": "",
+    "plugins": ["{{ rhsso_plugins | join('","') }}"],
```

Comment:
Could this be added to the keycloak cr in the operator repo so that it could be pulled in rather than needing to template it here? It could remove the need to template in the other installer too?

Reply:
We could, but not really sure if it's worth doing TBH. The main issue here is actually not the plugins, it's the name. In all the examples, they are using names like `examples`, which is not good to use here. Also the current version of the keycloak-operator is kind of deprecated. I would rather wait for the new version of the keycloak operator to be ready and then use that instead.